src/libs/Middleware/SaveResponseInOnyx.ts

```diff
@@ -10,6 +10,7 @@ const requestsToIgnoreLastUpdateID: string[] = [
     SIDE_EFFECT_REQUEST_COMMANDS.RECONNECT_APP,
     WRITE_COMMANDS.CLOSE_ACCOUNT,
     WRITE_COMMANDS.DELETE_MONEY_REQUEST,
+    WRITE_COMMANDS.SUBMIT_REPORT,
```

Comment:
@rafecolton @rojiphil @jacobkim9881 hey I noticed this change while investigating another issue. I don't have full context on this bug, but this is almost certainly the wrong way to fix it. We're essentially disabling reliable Onyx updates for `SubmitReport` requests here and ignoring any Onyx updates that may have been missed. I think we should revert this change and find another way to address the root problem.

Reply:
😬 thanks Andrew. I agree that does not sound great, and I did not realize that's what this change would do. Let's revert. I commented on the issue.

Reply:
No worries and thanks for raising the PR 🙇. Changes to this file aren't always obvious so maybe we should add a MelvinBot comment with a warning when a change is detected

Reply:
Yeah I think that would be a good improvement

Reply:
@arosiclair Thanks for reviewing. It makes me find an issue.

Reply:
> Thanks for reviewing. It makes me find an issue.

@jacobkim9881 Not sure if I understand. Can you please share the issue you found due to the change here?

Reply:
@rojiphil `getMissingOnyxUpdates` isn't activated when clicking submit button offline with the PR. It makes other user in the chat room couldn't get the result.

Reply:
> Thanks for reviewing. It makes me find an issue.

@rojiphil It is revealed as unrelated from my side.